test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -448,6 +466,20 @@
       (revoke-existing-cert! settings "localhost")
       (is (true? (revoked? cert))))))
 
+(deftest revoke-with-bundled-ca-certs
+  (testing "The CA certificate file can be a bundle when revoking a certificate"
+    (let [settings (testutils/ca-sandbox! cadir)
```

Comment:
Good catch, I copied too much from an existing test. Switching to use the bundle-cadir currently fails because we don't actually support CRL bundles; I'll fix that too.